modules/user_accounts/php/edit_user.class.inc

```diff
@@ -715,7 +715,8 @@ class Edit_User extends \NDB_Form
 
         // checking if the account can be rejected (no password hash
         // pending approval)
-        $this->tpl_data['can_reject'] =$this->_canRejectAccount($this->identifier);
+        $this->tpl_data['can_reject'] =
+            Edit_User::canRejectAccount($this->identifier);
```

Comment:
@driusan is this fine or should i be using self::canRejectAccount?

Reply:
self::canRejectAccount is probably better